.pre-commit-config.yaml

```diff
@@ -237,8 +237,7 @@ repos:
         entry: ./scripts/ci/pre_commit/check_deferrable_default.py
         pass_filenames: false
         # libcst doesn't have source wheels for all PY except PY3.12, excluding it
-        # libcst 1.8.1 doesn't include typing-extensions which is needed for Python 3.9
-        additional_dependencies: ['libcst>=1.1.0,!=1.8.0,!=1.8.1']
+        additional_dependencies: ['libcst>=1.1.0,!=1.8.0']
```

Comment:
```suggestion
        additional_dependencies: ['libcst>=1..8.1']
```

Reply:
Are you sure we should implicitly bump the lower version in the cleanup PR? (Assuming you wanted to propose `1.8.1` not `1..8.1`?)

Reply:
Yes. This cleans up things - libcst has very little dependencies:

```
Requires-Dist: pyyaml>=5.2; python_version < "3.13"
Requires-Dist: pyyaml-ft>=8.0.0; python_version >= "3.13"
Requires-Dist: typing-extensions; python_version < "3.10"
``` 

and we can bump ot to 1.8.1 (and yes I corrected it to remove ..)